auditlog/README.rst

```diff
@@ -11,6 +11,11 @@ Go to `Reporting / Audit / Rules` to subscribe rules. A rule defines which
 operations to log for a given data model.
 Then, check logs in the `Reporting / Audit / Logs` menu.
 
+By default logs older than 6 months (180 days) will be deleted automatically.
+To change this delay, go to the
+`Configuration / Technical / Automation / Scheduled Actions` menu and edit the
+`AutoVacuum audit logs` task parameters.
+
 During installation, it will migrate any existing data from the `audittrail`
```

Comment:
During installation, a one-time script will migrate any existing data from the `audittrail`
 module (rules and logs).